packages/ai/src/LLMStep.ts

```diff
@@ -148,6 +148,24 @@ export class LLMStepInstance<const Shape extends StepShape = StepShape> {
     }
   }
 
+  async run() {
+    this.log({
+      type: "info",
+      message: `Step "${this.template.name}" started`,
+    });
+
+    await this._run();
```

Comment:
Separation of concerns between `run` and `_run` is leaking here in several ways that make me think that `_run` should be inlined:
- "Step started" message above would happen even if `_run` decided not to run (see `this.state.kind !== "PENDING"` check at the start of `_run`)
- I don't think `this.state.kind !== "PENDING"` check below, in `completionMessage` construction, is needed — `_run` always ends with non-pending state